django/db/backends/oracle/operations.py

```diff
@@ -54,8 +54,8 @@ def autoinc_sql(self, table, column):
 DECLARE
     i INTEGER;
 BEGIN
-    SELECT COUNT(*) INTO i FROM USER_CATALOG
-        WHERE TABLE_NAME = '%(sq_name)s' AND TABLE_TYPE = 'SEQUENCE';
+    SELECT COUNT(1) INTO i FROM USER_SEQUENCES
```

Comment:
Oracle internally translates `count(1)` to `count(*)`, so they work exactly the same.